### DISCUSSION TITLE
Advice on rotating the screen

### DISCUSSION THREAD

**Discussion by wsnfly** [rustdesk/rustdesk#5448] [Ideas]
When using a computer to operate an Android phone, the Android phone often switches to landscape mode, but the computer cannot manually switch, which is sometimes inconvenient to operate with your neck tilted;)

**Comment by beelchester** [comment]
Do you mean something like this doesn't happen in your case?
(My android phone is in landscape mode here)
<img width="766" alt="Screenshot 2023-08-20 at 3 15 38 PM" src="https://github.com/rustdesk/rustdesk/assets/73148455/e9d36728-e16d-4bf7-8b1e-dc60cbb00210">
